src/extensions/default/InlineColorEditor/InlineColorEditor.js

```diff
@@ -97,25 +88,24 @@ define(function (require, exports, module) {
      * @return {?{start:{line:number, ch:number}, end:{line:number, ch:number}}}
      */
     InlineColorEditor.prototype.getCurrentRange = function () {
-        var start, end;
+        var pos, start, end;
         
-        start = this._startBookmark.find();
+        pos = this._marker.find();
```

Comment:
I just tested this - as long as the InlineColorEditor is open, `this._marker` is always an instance of `TextMarker`.